app/models/facility.rb

```diff
@@ -29,6 +29,7 @@ class Facility < ApplicationRecord
   has_many :patients, -> { distinct }, through: :encounters
   has_many :prescription_drugs
   has_many :appointments
+  has_many :teleconsultations
```

Comment:
Yes. Does it warrant being called `teleconsultation_requests`? I felt `teleconsultations` makes sense by itself since there is no concept of `teleconsultation_records` for a facility.

Reply:
Yup, the name is fine. I was just clarifying the semantics :)